DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -243,7 +243,7 @@
     process.SiStripAnalyserCosmic.TkMapCreationFrequency  = -1
     process.SiStripAnalyserCosmic.ShiftReportFrequency = -1
     process.SiStripAnalyserCosmic.StaticUpdateFrequency = 5
-    process.SiStripAnalyserCosmic.MonitorSiStripBackPlaneCorrection = False
+    process.SiStripAnalyserCosmic.MonitorSiStripBackPlaneCorrection = cms.bool(False)
```

Comment:
@jfernan2 the changes in this file are the only differences wrt the master version of this PR that I can't understand.
`MonitorSiStripBackPlaneCorrection` is defined, for example, in https://github.com/cms-sw/cmssw/blob/292e6ce4b6864ac1943e5442ee5a0b33089013f5/DQM/SiStripMonitorClient/python/SiStripClientConfigP5_cff.py#L39 : then what is the reason to add the explicit types here?

Reply:
This PR was forwardported to master in #35366 , not the other way around, and that's where I received comments to drop that type reconfiguring files in SiStripAnalyserCosmic 

I can put them in synch, despite both work, the master one is more optimal of course.

Reply:
Yes, please, try to sync them. I mean, only these explicit types, not the other code-format related changes.
We still have some time before cutting 12_0_2, and since you are responsible to sign this for dqm, I expect that it can be merged right after the automatic tests are concluded.

Reply:
> mean, only these explicit types, not the other code-format related changes

This is not possible, only a full synch works. The other changes were needed to drop those type specs

Anyway, I will try to synch asap

Reply:
> > mean, only these explicit types, not the other code-format related changes
> 
> This is not possible, only a full synch works. The other changes were needed to drop those type specs
> 
> Anyway, I will try to synch asap

Wait, if so we can let the backport PR as it is now.

However, I still don't understand why do you need to "drop those type specs": there is nothing to drop, because in the original 12_0_X configs the explicit types were not defined, and you only added them here. And given that `MonitorSiStripBackPlaneCorrection` is already defined in 12_0_X, as I pointed out above, I don't think (but I didn't try it...) that they are needed either. Could you please explain?

Reply:
Then, it must be a mistake from my side, you are right. Let me sync the two PRs so that we have both releases in agreement.
Thanks

Reply:
Done. In the process I found an addtional change needed in the just merged master PR https://github.com/cms-sw/cmssw/pull/35366
I will create a small PR for the modification. I apologize for the inconvenience. Thanks